### PR TITLE
Create psychologicalTesting.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
related to #283 

based on some more research I assume the "technique" behind checklists like CBCL, etc is better described with "psychological testing"

### CODE REVIEW

**Comment by UlrikeS91** [comment]
> related to #283
> 
> based on some more research I assume the "technique" behind checklists like CBCL, etc is better described with "psychological testing"

Sounds good. Would this mean that there should also be a technique called "physical testing"?

File: instances/technique/psychologicalTesting.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/psychologicalTesting",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Psychological testing' is the administration of psychological tests typically by trained evaluators which also evaluate the person's responses according to carefully prescribed guidelines resulting in psychometric scores are thought to reflect individual or group differences in the construct the test purports to measure. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Psychological_testing)]",
```
Comment (UlrikeS91) [inline instances/technique/psychologicalTesting.jsonld]: The definition is a bit long and confusing. 

Definition: "'Psychological testing' is a psychometric measurement to evaluate a person's response to a psychological test according to carefully prescribed guidelines. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Psychological_testing)]"
 
The rest could be moved into the description.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/technique/psychologicalTesting.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/psychologicalTesting.jsonld]: I like it. Thanks. And I updated it. Let's do a more detailed description later.

**Review by UlrikeS91 — APPROVED** [review]

File: instances/technique/psychologicalTesting.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/psychologicalTesting.jsonld]: but crashed the code... wait a moment

**Pull request merged**